int64 id
int64 id_a
int64 id_b
float64 value
bool routine
string group_name
int64[] trans_alphabet
float64 dtimes	
